nixos/modules/services/databases/victorialogs.nix

```diff
@@ -0,0 +1,114 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.victorialogs;
+  startCLIList = [
+    "${cfg.package}/bin/victoria-logs"
+    "-storageDataPath=/var/lib/${cfg.stateDir}"
+    "-httpListenAddr=${cfg.listenAddress}"
+  ] ++ cfg.extraOptions;
+in
+{
+  options.services.victorialogs = {
+    enable = mkEnableOption "VictoriaLogs is an open source user-friendly database for logs from VictoriaMetrics";
+    package = mkPackageOption pkgs "victoriametrics" { };
+    listenAddress = lib.mkOption {
+      default = "127.0.0.1:9428";
```

Comment:
Any reason to include 127.0.0.1 in this? The victoriametrics module uses the :<port> syntax

I don't have a preference either way, but it would be good to keep it consistent

Reply:
Yeah you're right, consistent is better.